pkg/apis/registries/v1alpha1/instance_types.go

```diff
@@ -56,6 +57,23 @@ type InstanceSpec struct {
 	Options *InstanceDeployOptions `json:"options,omitempty"`
 
 	HelmChart *InstanceHelmChartSpec `json:"helmChart"`
+
+	// +optional
+	GarbageCollection *GarbageCollectionReq `json:"garbageCollection,omitempty"`
+}
+
+// GarbageCollectionReq holds request information for a garbage collection schedule
+type GarbageCollectionReq struct {
+	Schedule *h.ScheduleParam `json:"schedule"`
+
+	// +optional
+	Name string `json:"name,omitempty"`
+	// +optional
+	Status string `json:"status,omitempty"`
+	// +optional
+	ID int64 `json:"id,omitempty"`
+	// +optional
+	Parameters map[string]byte `json:"parameters,omitempty"`
```

Comment:
A map of _single bytes_, really? Sure this wasn't supposed to be `map[string][]byte` or `map[string]string`?

Reply:
Should've been `map[string]string` :+1: